lib/top_hangman/io.rb

```diff
@@ -1,55 +1,81 @@
 module TopHangman
   class IO
-    class << self
-      def execute(game, renderer)
-        game.start
+    RUNNING = :running
+    STOPPED = :stopped
 
-        while game.state == Game::RUNNING
-          renderer.show_header(errors_count: game.current_round.errors_count)
+    def initialize
+      @state = RUNNING
```

Comment:
what is the purpose of this attribute? I think this `IO` class should be stateless, I mean, it shouldn't hold any attributes because that responsibility should be from `Game` or any other class. `IO` should call what needs to be called without holding anything

Reply:
This is intending to end the loop and doesn't execute any other method when the game is saved

<img width="437" alt="image" src="https://user-images.githubusercontent.com/26731448/195380624-69f68153-ef09-49a5-9608-5f4bd991ca48.png">

<img width="323" alt="image" src="https://user-images.githubusercontent.com/26731448/195380656-f2484d89-554e-484f-a002-a414a0bb5004.png">

Reply:
Can you use the `game.state` for this, instead of this `state`?

Reply:
done!